ui/app/routes/vault/cluster/access/oidc/clients/client/providers.js

```diff
@@ -3,6 +3,7 @@ import Route from '@ember/routing/route';
 export default class OidcClientProvidersRoute extends Route {
   model() {
     const model = this.modelFor('vault.cluster.access.oidc.clients.client');
+    // TODO CMB: broken - waiting for backend to accept GET + ?list=true
```

Comment:
I've un-done the Jira ticket related to this work, and linked the backend ticket that's currently blocking it.